### ISSUE TITLE
ActiveMQ.DLQ memory usage

### ISSUE THREAD

**Issue by tkohegyi** [epam/Wilma#55]
In case of extra large messages (think about >60 Mbytes per msg), and high load it may happen that ActiveMQ runs out of memory, like this:
```
15:17:31.447 [reponseJmsContainer-2] INFO  c.e.w.core.jms.LoggerErrorHandler - Error occurred in response queue: 
java.lang.OutOfMemoryError: Requested array size exceeds VM limit
15:17:39.657 [reponseJmsContainer-2] INFO  c.e.w.core.jms.LoggerErrorHandler - Error occurred in response queue: 
java.lang.OutOfMemoryError: Requested array size exceeds VM limit
```
And this can cause moving that message to the DQL. If this happens many times, DLQ size is increasing and can eat up all the available memory, and can result of high load, then ActiveMQ failure then run out of threads in Jetty, and finally Wilma cannot handle incoming requests. This sequence of problems may take hours so Wilma can protect itself against it, by cleaning up the DLQ.  

This ticket is partially addressed by commit 193e0a023d718835c0c52e96184419afe90139cf. so will be part of the next release soon.

**Comment by tkohegyi** [comment]
Testing is ongoing, if everything is ok, a new Wilma release will be published.

**Comment by tkohegyi** [comment]
Test passed successfully. 
```
13:10:12.375 [reponseJmsContainer-9] INFO  c.e.w.core.jms.LoggerErrorHandler - Error occurred in response queue: 
java.lang.OutOfMemoryError: Requested array size exceeds VM limit
13:10:15.008 [RMI TCP Connection(125)-10.208.10.32] INFO  o.a.activemq.broker.region.Queue - queue://ActiveMQ.DLQ purged of 1 messages
13:10:15.008 [maintenanceScheduler-4] INFO  c.e.w.s.monitor.JmsQueueMonitorTask - Message found in ActiveMQ.DLQ. Queue size is: 1, queue purged successfully.
```
New Wilma release is coming soon.

**Comment by tkohegyi** [comment]
Published with V1.1.73
